modules/fedimint-ln-common/src/api.rs

```diff
@@ -59,6 +59,9 @@ where
         .await
     }
 
+    /// There is no consensus within Fedimint on the gateways, each guardian
+    /// might be aware of different ones, so we just return the union of all
+    /// responses and allow client selection.
```

Comment:
Could be more succinct, only describing what it does, i.e. "returns union of gateways registered with guardian servers". Why that's the case is less relevant for the function's documentation (that's the caller's problem) and should thus only be mentioned further down if at all.